apps/give-base/src/components/carbonchart.tsx

```
@@ -0,0 +1,101 @@
+  const dec = Number(fix) * max
+  const prt:any = pct[dec]
+  const rst = max - int - ext
+  console.log('Tons', num, mod, dec, rst)
+  const offs = Array(int).fill(0)
+  const tons = Array(rst).fill(0)
+  //console.log('Arrs', offs.length, tons.length)
+  return (
+    <>
+      <div className="text-center mb-4">{title}</div>
+      <div style={style.chart}>
+        {offs.map(() => { return (<div style={style.off} key={Math.random()}></div>) } )}
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)